src/pages/event/[id].tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router";
+
+const EventDetail: React.FC = () => {
+  const router = useRouter();
+  const { id, name, sport } = router.query;
+
+  return (
+    <div>
+      <p>Name: {name}</p>
+      <p>Sport: {sport}</p>
+      <p>Id: {id}</p>
+      <button onClick={() => router.push("/")}>Back</button>
+    </div>
+  );
```

Comment:
Fetch the details here from the /{id} endpoint. Dont pass it like this. Only the ID

Reply:
resolved